index.js

```diff
@@ -5,10 +5,10 @@ module.exports = function (rgbString) {
 	// First remove all whitespace, then parse the rgb string
 	rgbString = rgbString.replace(/ /g, '');
 
-	rgbString.replace(/rgb\((\d+),(\d+),(\d+)\)/, function (rgbString, r, g, b) {
-		rgb.red = Number(r);
-		rgb.green = Number(g);
-		rgb.blue = Number(b);
+	rgbString.replace(/rgb\((\d+|-\d+),(\d+|-\d+),(\d+|-\d+)\)/, function (rgbString, r, g, b) {
```

Comment:
The only reason, I still learn regular expressions and not strong enough in it :)